app/src/main/java/org/wikipedia/page/PageTitle.java

```diff
@@ -232,11 +232,12 @@ public void setText(@NonNull String convertedFromText) {
     }
 
     @NonNull public String getDisplayText() {
-        return displayText == null ? StringUtil.removeUnderscores(getPrefixedText()) : displayText;
+        return displayText == null ? StringUtil.removeUnderscores(getPrefixedText())
```

Comment:
Sorry, this is starting to cause underlying issues for titles that have a colon in their name, for example:

![image](https://user-images.githubusercontent.com/1682214/108111976-7744c180-7063-11eb-8c9c-b71b3f1e6851.png)

Can we solve your issue without making any changes to the `getDisplayText` and `setDisplayText` functions?

Reply:
Sure, I'll go ahead to make another solution for the issue.

Reply:
Looks like the latest version removes the "Star trek:" in the talk page. The new commit fixes it but maybe we should consider updating the `removeNamespace()` or making another one like `removeWikiNamespace()`. 
![Screenshot_20210216-152616_Wikipedia](https://user-images.githubusercontent.com/2435576/108135060-4452fb80-706c-11eb-9ae1-738caf655dab.jpg)